app/assets/stylesheets/jquery-ui/custom-theme/jquery-ui-1.8.13.custom.scss

```diff
@@ -291,8 +281,8 @@
 .ui-corner-all { -moz-border-radius: 4px; -webkit-border-radius: 4px; border-radius: 4px; }
 
 /* Overlays */
-.ui-widget-overlay { background: #aaaaaa image_path('ui-bg_flat_0_aaaaaa_40x100.png') 50% 50% repeat-x; opacity: .30;filter:Alpha(Opacity=30); }
-.ui-widget-shadow { margin: -8px 0 0 -8px; padding: 8px; background: #aaaaaa image_path('ui-bg_flat_0_aaaaaa_40x100.png') 50% 50% repeat-x; opacity: .30;filter:Alpha(Opacity=30); -moz-border-radius: 8px; -webkit-border-radius: 8px; border-radius: 8px; }/*
+.ui-widget-overlay { background: #aaaaaa ; opacity: .30;filter:Alpha(Opacity=30); }
```

Comment:
Color literals like `#aaaaaa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.30` should be written with a leading zero as `0.30`<br>Merge rule `.ui-widget-overlay` with rule on line 41<br>Name of function `Alpha` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered background, filter, opacity<br>Colon after property should be followed by one space<br>`Opacity=30` should be written with a single space on each side of the operator: `Opacity = 30`<br>Declaration should not have a space before the terminating semicolon